cleanlab/object_detection/rank.py

```diff
@@ -448,7 +467,8 @@ def _get_valid_inputs_for_compute_scores_per_image(
             if 0 in similarity_matrix.shape
             else np.min(similarity_matrix[np.nonzero(similarity_matrix)])
         )
-
+    if has_overlap_label_bboxes is None:
```

Comment:
Should the user have an option to disable this?

Reply:
yes ideally there's way for use to ignore this sort of issue

Reply:
Added : https://github.com/cleanlab/cleanlab/blob/401743a946885e7a29a2cea4df48e761b60c05a1/cleanlab/object_detection/rank.py#L109